app/views/shared/_cookie_banner.html

```diff
@@ -0,0 +1,40 @@
+<div
+  id="cookie-banner"
+  class="cookie-banner"
+  style="display: none"
+>
+  <h2
+    class="cookie-banner__heading"
+  >
+    This site uses cookies
+  </h2>
+  <p
+    class="cookie-banner__details"
+  >
+    Cookies are files stored on your computer, phone, or tablet.
+    They help us understand how you use this site, like which pages you visit.
+    <br/>
+    For more information, view our
+      <a
+        href="#"
```

Comment:
I'll ask Dave L where we should point this

Reply:
Will remove the link